direct/tpm2/reflect.go

```diff
@@ -20,7 +20,7 @@ import (
 const (
 	// Chosen based on MAX_DIGEST_BUFFER, the length of the longest
```

Comment:
Can you update this comment explaining that maxListLength needs to be greater than MAX_CONTEXT_SIZE, which in the reference implementation is 1344?

Reply:
Updated!